webviz_ert/assets/ert-style.json

```diff
@@ -131,6 +131,18 @@
                 "y": -0.3,
                 "x": 0.5
             }
+        },
+        "layout-date-value-axis-labels": {
+            "xaxis": {
+              "title": {
+                "text": "Date"
```

Comment:
The x_axis is not always a datetime, in the poly_example the index is just a numerical index. Is it possible to change the label based on the index type? `Date` for datetime and maybe just `Index` for numerical values. 

Or maybe somebody has a better idea than `Index` @mortalisk @frode-aarstad @oysteoh

Reply:
I'll look into it and see if I can access the necessary info! thanks for pointing out the example

Reply:
so looking at the code, I can see where I can access the info - the `response` of type `webviz.models.Response` has a property `data` that contains a panda dataframe. the `index` of the dataframe has a type - in the case of poly_example, the type is `pandas.core.indexes.numeric.Int64Index`, but for snake_oil_field, the type is just a generic `pandas.core.indexes.base.Index`.

a possible heuristic at this point would be to set a label "Index" when the type is Int64Index, and "Date" otherwise.

one thing that makes me hesitate with implementing this is that the properties i need are not quite readily available / accessible where I want to use them. maybe i don't quite see how i should use the code, maybe it needs a bit of restructuring - in any case, i would love some input on this, maybe we could have a chat about this tomorrow?

Reply:
after a live discussion, Dan pointed out an elegant solution to my problem.

now all that's left is to add some test(s) that cover this - it seems like i will need to add some test data in `tests/data/snake_oil_data.py` ... too late to try and wrap my head around that now

Reply:
resolving this as the label is dynamic now